src/test/java/uk/gov/hmcts/reform/profilesync/schedular/UserProfileSyncJobSchedulerTest.java

```diff
@@ -2,85 +2,115 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 import java.time.LocalDateTime;
-
-import org.junit.Before;
 import org.junit.Test;
 import uk.gov.hmcts.reform.profilesync.advice.UserProfileSyncException;
-import uk.gov.hmcts.reform.profilesync.domain.SyncJobAudit;
+import uk.gov.hmcts.reform.profilesync.domain.ProfileSyncAudit;
 import uk.gov.hmcts.reform.profilesync.domain.SyncJobConfig;
+import uk.gov.hmcts.reform.profilesync.repository.ProfileSyncAuditRepository;
 import uk.gov.hmcts.reform.profilesync.repository.SyncConfigRepository;
-import uk.gov.hmcts.reform.profilesync.repository.SyncJobRepository;
 import uk.gov.hmcts.reform.profilesync.service.ProfileSyncService;
 
 public class UserProfileSyncJobSchedulerTest {
+    //mocked as its an interface
```

Comment:
I don't think this comment is required anymore if you removed `syncJobRepository`

Reply:
ProfileSyncAuditRepository also interface so comment is valid one only